bundleserver/src/main/java/net/discdd/server/bundletransmission/ServerBundleTransmission.java

```diff
@@ -258,6 +258,7 @@ public GetRecencyBlobResponse getRecencyBlob(String senderId) throws InvalidKeyE
         byte[] publicKeyBytes = this.bundleSecurity.getIdentityPublicKey();
         return GetRecencyBlobResponse.newBuilder()
                 .setStatus(RecencyBlobStatus.RECENCY_BLOB_STATUS_SUCCESS)
+                .setMessage("this is a test for the message")
```

Comment:
since you have the message in the repository now, you should pull it from there rather than hardcoding.